src/acq/adc.c

```diff
@@ -655,7 +655,9 @@ static void bl_acq_adc_dma_isr(bl_acq_adc_t *adc, unsigned buffer)
 		if (bl_led_src[bl_led_active].src_mask & adc->config.src_mask) {
 			bl_acq_channel_commit_sample(
 					adc->config.channel_index[c], sample[c]);
-			bl_led_loop();
+
+			if (bl_led_count > 1)
+				bl_led_loop();
```

Comment:
Code style requires curly braces around this if statement.

Reply:
+1